### ISSUE TITLE
Parts of the code could be improved in version 1.3.0

### ISSUE THREAD

**Issue by karmacoma** [LiskHQ/lisk-mobile#1036]
Parts of the code that could be improved: ● There are no env variables - lack of possibility to release for multiple environments and versions. Probably it requires changing config in codebase which is not elegant and bad practice ● 0.59 React Native version is used without support of Android X. Upgrading React Native version may be painful. ● No strong typing with typescript ● Thunk as middleware. Usage of Saga or Observable is preferred. ● No styled components. Only default React Native way of styling ● Some components could be more granular. 400 lines for single component may be to much to have good readability

It’s also a good practice to add ​quality badges​ to open source projects that attract users to use Lisk projects and open source contributors to contribute.

### Tasks

- Update react native version to the latest version #1003
- All components should have less than 400 lines of code #1029
- Update the repository badges #1041

**Comment by karmacoma** [comment]
All tasks were done.

**Comment by yasharAyari** [comment]
● when we start the lisk-mobile we chose not to use typescript in lisk-moble to be aligned with lisk-desktop. In our last governance meeting, we decided not to use Typescript in lisk-mobile. ● I don't see any benefits in using style-component in lisk-mobile and it just adds extra complexity to our code-base. ● redux-thunk fulfill all of our needs in lisk-mobile and redux-saga is too much unnecessary complexity for a small project like lisk-mobile.
